### DISCUSSION TITLE
Docker image available?

### DISCUSSION THREAD

**Discussion by LukeEvansTech** [danricho/where_the#20] [Q&A] [answered]
Hi all,

I'm wondering if I have just missed this, but is the Docker image published to GitHub Container Registry or Docker Hub at all?

Would prefer to be able to pull this image down rather than pulling updated files from GitHub if possible.

Thanks!

**Comment by danricho** [comment]
Hi,

Not at this stage. 
I’ve never published such an image so I can’t see it happening any time soon.
I do try to make sure updates won’t be breaking (as much as reasonable).

cheers!

**Reply by danricho** [reply]
That being said, if someone figured out an appropriate GitHub action for the image to be built and published on each commit I’d be interested!

Eventually I’ll get time and have a go… but can’t commit to when.

for my reference:
https://docs.github.com/en/actions/publishing-packages/publishing-docker-images

**Comment by danricho** [comment, marked as answer]
Well with your help, Yes there is!!
Thanks again Luke.
